### PR TITLE
LPS-150951 [CP 2.0] Fix button "add activation key" is disabled

### PR DESCRIPTION
**Pull request by joaosvassis**

### CODE REVIEW

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js [hunk 1]
```diff
@@ -46,6 +46,9 @@ const RequiredInformation = ({
 	const hasTouched = !Object.keys(touched).length;
 	const hasError = Object.keys(errors).length;
 
+	// eslint-disable-next-line no-console
+	console.log(`infoSelectedKey=`, infoSelectedKey);
+
```
Comment (joanagrego) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js]: I think that you forgot to delete this.

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js [hunk 2]
```diff
@@ -46,6 +46,9 @@ const RequiredInformation = ({
...
 	const avaliableKeysMaximumCount =
 		infoSelectedKey.selectedSubscription?.quantity;
```
Comment (joanagrego) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js:50]:
```suggestion
		infoSelectedKey?.selectedSubscription?.quantity;
```

**Review by joanagrego — APPROVED** [review]
Nice!

**Review by Lucas2691 — APPROVED** [review]

File: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js [hunk 3]
```diff
@@ -113,8 +113,7 @@ const RequiredInformation = ({
 			);
 
 			setShowKeyEmptyError(true);
-		}
-		else {
+		} else {
```
Comment (iagochaves) [inline modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/GenerateNewKey/pages/RequiredInformation.js:116]: Run SourceFormat here

**Review by iagochaves — APPROVED** [review]
Nice!

**Review by igordmf — APPROVED** [review]
Nice job!

**Review by jairmedeiros — APPROVED** [review]

**Comment by jairmedeiros** [comment]
ci:forward

**Pull request closed without merging**